gravitee-apim-e2e/package.json

```diff
@@ -9,10 +9,9 @@
         "build": "tsc --removeComments --outDir dist --target es5",
         "bulk": "jest -c jest.config.bulk.js -e --forceExit --logHeapUsage --useStderr --runInBand --detectOpenHandles",
         "test:ui": "sh ./scripts/run-docker-compose.sh ui-test mongo",
-        "test:ui:dev": "cypress open --config-file ui-test/integration/apim/ui/cypress-config.json",
-        "test:ui:bulk": "cypress run --config-file ui-test/integration/apim/bulk/cypress-config.json",
```

Comment:
why did you delete this line ?

Reply:
Ok, well spotted. Not sure how this happened, it was not done intentionally. I'll bring it back.

Reply:
A little update on this:
It seems that the bulk file was once migrated and is now to be executed by Jest. In fact, there's a npm run script for bulk in line 10 (as seen in your excerpt of the package.json):
`"bulk": "jest -c jest.config.bulk.js -e --forceExit --logHeapUsage --useStderr --runInBand --detectOpenHandles",`

Which means that the `test:ui:bulk` entry is actually obsolete and I'll leave it as it is now.